content/en/docs/platforms/kubernetes/operator/_index.md

```diff
@@ -29,8 +29,8 @@ The operator manages:
 
 ## Getting started
 
-To install the operator in an existing cluster, make sure you have cert-manager
-installed and run:
+To install the operator in an existing cluster, make sure you have
+[`cert-manager` installed](https://cert-manager.io/docs/installation/) and run:
```

Comment:
With the monospace formatting on cert-manager, the link has a weird break in it in the rendered version. So I think we can leave "installed" out of the link text in this case.

```suggestion
[`cert-manager`](https://cert-manager.io/docs/installation/) installed and run:
```